ratis-test/src/test/java/org/apache/ratis/datastream/DataStreamTestUtils.java

```diff
@@ -154,7 +154,7 @@ class MultiDataStreamStateMachine extends BaseStateMachine {
     @Override
     public CompletableFuture<DataStream> stream(RaftClientRequest request) {
       final SingleDataStream s = new SingleDataStream(request);
-      LOG.info("XXX {} put {}, {}", this, ClientInvocationId.valueOf(request), s);
```

Comment:
I optimized the logs by removing the `XXX` prefix. I understand it was likely added for debugging purposes, but such prefixes are not appropriate in a production environment.